### DISCUSSION TITLE
Feature request: Is it possible to add a time for how long  are confetti visible before fading out?

### DISCUSSION THREAD

**Discussion by jakubsobotkatv** [simibac/ConfettiSwiftUI#27] [Ideas]
Feature request: Is it possible to add a time for how long  are confetti visible before fading out?

I need the time to be shorter but I don't know how to do it. I guess there is not parameter for it. 
Would be super awesome!

`.confettiCannon(
                            counter: $conffetiErrorTrigger,
                            num: 6,
                            confettis: [.text("❌"), .shape(.circle)],
                            colors: [.red],
                            confettiSize: 20.0,
                            rainHeight: 600,
                            openingAngle: Angle(degrees: 0),
                            radius: 600
                            )`

Anyway, thank for the great Confetti UI.

**Comment by simibac** [comment]
Hi, thanks for reaching out. There is no parameter to do that rn. I am quite busy rn to do it myself. You probably need to dispatch the fading out animation separately with a different animation curve here: https://github.com/simibac/ConfettiSwiftUI/blob/master/Sources/ConfettiSwiftUI.swift#L229

**Reply by simibac** [reply]
https://cubic-bezier.com/#.75,-0.01,.24,1.62

Tools like this can help visualize the animation
